bool coordinates_demand
---
float64 x_pose
float64 y_pose
float64 z_pose

float64 x_orientation
float64 y_orientation
float64 z_orientation
float64 w_orientation

float64 nx
float64 ny
float64 nz

int32 remaining_objects
